src/app/api/settings/route.ts

```diff
@@ -0,0 +1,46 @@
+import { PrismaClient } from '@prisma/client'
+
+const prisma = new PrismaClient()
+
+export async function POST(request: any) { 
+    try {
+        const { host, port, secure, username, password } = await request.json();
+        const newSetting = await prisma.mailSetting.upsert({
+            where: {
+              username: 'arina@codetisan.com',
+              
+            },
+            update: {
+                host,
+                port,
+                secure,
+                username,
+                password,
```

Comment:
Is there a way to encrypt the password? So that we don't store the plain text password. If the DB is compromised, the SMTP account will also leak, which is not good.